### PR TITLE
DataWeave In Apex Basic Recipes

### PR DESCRIPTION
**Pull request by msrivastav13**
### What does this PR do?

Adds basic DataWeave In Apex recipes.

### What issues does this PR fix or reference?

Adds basic DataWeave In Apex recipes. The recipes are HelloWorld examples. More relevant recipes will be added as well in next PR.

## The PR fulfills these requirements:

[X ] Tests for the proposed changes have been added/updated.
[X ] Code linting and formatting was performed.

### CODE REVIEW

File: force-app/main/default/classes/DataWeaveInApex Recipes/BasicDataWeaveInApexRecipes.cls [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/**
+ * @description Demonstrates various ways of invoking a DataWeave scripts
+ * in apex. Note that this class demonstrates both static and dynamic way of
+ * invoking DataWeave scripts .
```
Comment (pozil) [inline force-app/main/default/classes/DataWeaveInApex Recipes/BasicDataWeaveInApexRecipes.cls:4]:
```suggestion
 * invoking DataWeave scripts. For more DataWeave for Apex example head to https://github.com/developerforce/DataWeaveInApex.
```

File: force-app/main/default/classes/DataWeaveInApex Recipes/BasicDataWeaveInApexRecipes.cls [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/**
+ * @description Demonstrates various ways of invoking a DataWeave scripts
+ * in apex. Note that this class demonstrates both static and dynamic way of
```
Comment (pozil) [inline force-app/main/default/classes/DataWeaveInApex Recipes/BasicDataWeaveInApexRecipes.cls:3]:
```suggestion
 * in Apex. Note that this class demonstrates both static and dynamic way of
```

File: force-app/main/default/staticresources/documentation/CsvToJsonConversion.md
Comment (msrivastav13) [inline force-app/main/default/staticresources/documentation/CsvToJsonConversion.md:1]: @pozil FYI this recipe I plan to add in upcoming commit. Still working on test cases.

File: force-app/main/default/staticresources/documentation/CsvToJsonConversion.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# CsvToJsonConversion
```
Comment (pozil) [inline force-app/main/default/staticresources/documentation/CsvToJsonConversion.md:1]: This file doesn't belong here. It is likely a leftover from a previous apexdocs execution. I recommend that you `rm force-app/main/default/staticresources/documentation/*` and re-run apexdocs.

File: force-app/tests/DataWeaveInApex Recipes/BasicDataWeaveInApexRecipes_Tests.cls [hunk 4]
```diff
@@ -0,0 +1,39 @@
+@isTest
+public class BasicDataWeaveInApexRecipes_Tests {
+    @isTest
+    static void testStaticInvocation() {
+        Test.startTest();
```
Comment (pozil) [inline force-app/tests/DataWeaveInApex Recipes/BasicDataWeaveInApexRecipes_Tests.cls:5]: Do we need this for DataWave only execution?

**Review by pozil — COMMENTED** [review]
Looks like we're missing `package.json` and there are some issues with docs (see comments).

File: force-app/tests/DataWeaveInApex Recipes/BasicDataWeaveInApexRecipes_Tests.cls (hunk 4 above)
Comment (msrivastav13) [reply force-app/tests/DataWeaveInApex Recipes/BasicDataWeaveInApexRecipes_Tests.cls:5]: You mean `Test.startTest()` here?

Comment (msrivastav13) [reply force-app/tests/DataWeaveInApex Recipes/BasicDataWeaveInApexRecipes_Tests.cls:5]: Its generally a best practice to have a  `Test.startTest()` and `Test.stopTest()` to reset the governor limit.

**Review by pozil — APPROVED** [review]
Looking good!

**Pull request merged**